automation/taskcluster/androidTest/flank-arm64-v8a.yml

```diff
@@ -44,10 +48,6 @@ flank:
   # test shards - the amount of groups to split the test suite into
   # set to -1 to use one shard per test.
   max-test-shards: -1 
-  # repeat tests - the amount of times to run the tests.
+  # num-test-runs- the amount of times to run the tests.
```

Comment:
I changed this param because we got a warning from the Firebase API that this param had been replaced with the new one specified here.